MainController#date_select_format? check params['timecop'] first

ferndopolis opened the pull request:
I discovered that when you pass a date hash not using the timecop format to the update controller you get the following error:

NoMethodError: undefined method `[]' for nil:NilClass

Solution is to check first for the existence of params['timecop'] before getting into the nested hash.

ferndopolis committed:
spec for backward compatible params set to MainController#update

ferndopolis committed:
check existence of params[timecop] first

jtrupiano commented:
Thanks for the contribution.  I'm no longer maintaining this project though.

ferndopolis commented:
how would yall feel if started maintaining the project?  a friend and i have a few feature we'd like to add that are super useful.

do you use any another gem like this one for using timecop in development?

jtrupiano commented:
No, I'm not aware of another project like this.  I'm happy to hand the reins over to you.  Shall I transfer the project over to your GH account?

ferndopolis commented:
yeah let's do it, thanks!

jtrupiano commented:
I've added you as a collaborator.  It won't let me transfer ownership while you have your own fork on GH.  So go ahead and merge your change over to jtrupiano/timecop-console, then delete your fork.  After that I can transfer ownership.

ferndopolis commented:
ok

On Thu, Sep 5, 2013 at 10:48 AM, John Trupiano notifications@github.comwrote:

> I've added you as a collaborator. It won't let me transfer ownership while
> you have your own fork on GH. So go ahead and merge your change over to
> jtrupiano/timecop-console, then delete your fork. After that I can transfer
> ownership.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/jtrupiano/timecop-console/pull/3#issuecomment-23887658
> .

ferndopolis commented:
i deleted my fork, should be ready to transfer

On Thu, Sep 5, 2013 at 11:11 AM, Fernando Marcheselli <
fmarcheselli@gmail.com> wrote:

> ok
> 
> On Thu, Sep 5, 2013 at 10:48 AM, John Trupiano notifications@github.comwrote:
> 
> > I've added you as a collaborator. It won't let me transfer ownership
> > while you have your own fork on GH. So go ahead and merge your change over
> > to jtrupiano/timecop-console, then delete your fork. After that I can
> > transfer ownership.
> > 
> > —
> > Reply to this email directly or view it on GitHubhttps://github.com/jtrupiano/timecop-console/pull/3#issuecomment-23887658
> > .
